vignettes/A_Honeybee_biology.Rmd

```diff
@@ -85,56 +83,55 @@ founderGenomes2 <- simulateHoneyBeeGenomes(nMelN = 4,
 ```
 
 Unfortunately, due to the complexity of this function, even using such small
-numbers requires a lot of computing power and takes a while to run. Simulating a
-group of founder genomes with more realistic numbers will therefore require a
-lot of time to run. We suggest outsourcing this process to an external server
-and to save the outcome as an Rdata file. This Rdata file is easy to load into
-your environment and work with.
+numbers takes a while to run. Simulating a group of founder genomes with more
+realistic numbers will therefore require a lot of time to run. We suggest
+running this part to an external server and save the outcome as an RData file,
+which we can load in our environment and work with it.
 
 ```{r save Rdata file}
 save(founderGenomes2, file="FounderGenomes2_3chr.RData")
 ```
 
 Besides specifying the number of individuals, chromosomes, and segregating
 sites, `simulateHoneybeeGenomes()`, also takes a number of genomic parameters:
-effective population size, ploidy, number of base pairs in a chromosome, genetic
-length of a chromosome, mutation rate, and recombination rate. The default
-values for these numbers follow published references, hence we do not recommend
-changing them, although you can. Also, the effective population size refers to
-the Wallberg demographic model (Wallberg et al., 2014) that is internally used
-for simulating the honeybee demography. Thus, this value cannot be changed
-safely and would not recommend its change.
+effective population size, ploidy, length of chromosomes in base pairs, genetic
+length of a chromosome in Morgans, mutation rate, and recombination rate. The
+default values for these numbers follow published references (Wallberg et al.,
+2014). You can read more about these in the help page:
```

Comment:
I would still briefly mention here that changing these numbers is not safe (especially Ne)

Reply:
Yes, this is important!